flask_saml_sso/sso.py

```diff
@@ -124,23 +134,36 @@ def api_token():
 
     If no user is currently logged in, redirect to SSO flow and return here
     """
+    logger.info('API-token called')
```

Comment:
I think this is a bit too much for the `INFO` level, but perhaps useful for `DEBUG`. Wouldn't it be better if `INFO` merely logged the _result_ of each invocation?

Reply:
What do you mean by result?

Reply:
Ideally, we'd `INFO` one line for each request, and that line would tell you everything™ you need to know. `X called` seems like `DEBUG` to me.